log/stdlib.go

```diff
@@ -39,9 +39,9 @@ func TimestampKey(key string) StdlibAdapterOption {
 	return func(a *StdlibAdapter) { a.timestampKey = key }
 }
 
-// FileKey sets the key for the file and line field. By default, it's "file".
-func FileKey(key string) StdlibAdapterOption {
-	return func(a *StdlibAdapter) { a.fileKey = key }
+// CallerKey sets the key for the file and line field. By default, it's "caller".
+func CallerKey(key string) StdlibAdapterOption {
```

Comment:
To preserve backwards compatibility, could we leave the name of the option as `FileKey`?